languages/elixir/exercises/concept/charlists/lib/username.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Username do
+  def sanitize(username) do
+    # ä -> ae
+    # ö -> oe
+    # ü -> ue
+    # ß -> ss
```

Comment:
Maybe rather than hinting at the code syntax, use words?

```suggestion
    # ä becomes ae
    # ö becomes oe
    # ü becomes ue
    # ß becomes ss
```